src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2902,7 +2902,7 @@ components:
           type: string
         cloudPlatform:
           $ref: '#/components/schemas/CloudPlatform'
-        secureMonitoringEnabled:
+        enableSecureMonitoring:
```

Comment:
I forget, it this same name of the property that gets returned?  It should probably be symetrical

Reply:
Its not the same property. The property in the request describes an action, while the property in the response describes the state of a dataset/snapshot.

Reply:
Ah right...fair.  It's not very RESTful (embedding actions in a payload like that) but I get this is a bit of an interesting case